casper/src/main/scala/coop/rchain/casper/util/comm/ListenAtPar.scala

```diff
@@ -0,0 +1,28 @@
+package coop.rchain.casper.util.comm
+
+import coop.rchain.models.GUnforgeable.UnfInstance.{GDeployIdBody, GDeployerIdBody, GPrivateBody}
+import coop.rchain.models.syntax.modelsSyntaxString
+import coop.rchain.models.{GDeployId, GDeployerId, GPrivate, GUnforgeable, Par}
+
+object ListenAtPar {
+  sealed trait Name
+  final case class UnforgPrivate(data: String)  extends Name
+  final case class UnforgDeploy(data: String)   extends Name
+  final case class UnforgDeployer(data: String) extends Name
+
+  def create(name: String, data: String): Option[Name] =
+    name match {
+      case "UnforgPrivate"  => Some(UnforgPrivate(data))
+      case "UnforgDeploy"   => Some(UnforgDeploy(data))
+      case "UnforgDeployer" => Some(UnforgDeployer(data))
+      case _                => None
+    }
+
+  def toPar(name: Name): Par = Par(unforgeables = Seq(GUnforgeable(unforgToUnforgProto(name))))
+
+  private def unforgToUnforgProto(name: Name): GUnforgeable.UnfInstance = name match {
+    case UnforgPrivate(data)  => GPrivateBody(GPrivate(data.unsafeHexToByteString))
+    case UnforgDeploy(data)   => GDeployIdBody(GDeployId(data.unsafeHexToByteString))
+    case UnforgDeployer(data) => GDeployerIdBody(GDeployerId(data.unsafeHexToByteString))
+  }
+}
```

Comment:
Please split CLI part as a separate PR that we can finish later. We don't want to add additional command that doesn't work from the start.